fabric/sgxconfig/ias/.gitignore

```diff
@@ -0,0 +1,3 @@
+client.crt
```

Comment:
needed?
I guess we can let the user choose environment variables for these, and keep them somewhere else in the system

Reply:
given that this is the default location and it must be user-provided (the original dummy one didn't work) i'd rather put it into .gitignore